documentation/schema.yaml

```diff
@@ -968,6 +973,27 @@ referencedSchemas:
           type: boolean
         type:
           type: string
+  Funding:
+    required:
+    - type
+    type: object
+    properties:
+      source:
+        type: string
+        format: uri
+      title:
```

Comment:
We can have titles in more than one language can we not?

Reply:
The UI design will not support it, but we can add it and assume Norwegian?

Reply:
Or use mainTitle and provided ways of setting alternative titles in a map like for publication?

Reply:
We can still have a language map in Cristin proxy, and then the user of the proxy can choose which language themselves?

Reply:
Note that for standalone fundings for registrations that I am working on, we don't store title for funding sources. Only for the project: 
![Screenshot 2022-12-09 at 14 36 50](https://user-images.githubusercontent.com/55836988/206714462-67dabe23-5a6c-422e-9ee7-c448c00b1cfc.png)

Reply:
Hmm, do we have to write these to Cristin so they can be reused for different publications?